dolphinscheduler-registry/dolphinscheduler-registry-plugins/dolphinscheduler-registry-jdbc/src/main/java/org/apache/dolphinscheduler/plugin/registry/jdbc/server/JdbcRegistryServer.java

```diff
@@ -303,12 +303,18 @@ private void doPurgeJdbcRegistryClientInDB(final List<Long> jdbcRegistryClientId
             return;
         }
         log.info("Begin to delete dead jdbcRegistryClient: {}", jdbcRegistryClientIds);
-        jdbcRegistryDataRepository.deleteEphemeralDateByClientIds(jdbcRegistryClientIds);
+        deleteEphemeralDataByClientIds(jdbcRegistryClientIds);
```

Comment:
We should remove in this method, we only need to delete the jdbcRegistryClient here.
```java
jdbcRegistryDataRepository.deleteEphemeralDateByClientIds(jdbcRegistryClientIds);
jdbcRegistryLockRepository.deleteByClientIds(jdbcRegistryClientIds);
```